app/services/fact_parser.rb

```diff
@@ -61,10 +81,14 @@ def has_comment?
   end
 
   def hostgroup
-    hostgroup_title = facts[:foreman_hostgroup]
+    hostgroup_title = hostgroup_name
     Hostgroup.unscoped.where(:title => hostgroup_title).first_or_create if hostgroup_title.present?
   end
 
+  def hostgroup_name
```

Comment:
@ofedoren suggested this in the related PR:
```suggestion
  def hostgroup_title
```

I'm not sure which is best, but it shouldn't be lost.

Reply:
Well title seems better fit for me.